### ISSUE TITLE
2021-04

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#15]

**Comment by qiaojy19** [comment]
# Apr - 2021

## 04.26 - 04.30
| Title                                                                                                                                                            |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Towards Fair Federated Learning with Zero-Shot Data Augmentation](https://arxiv.org/abs/2104.13417)                                                             |
| [Cluster-driven Graph Federated Learning over Multiple Domains](https://arxiv.org/abs/2104.14628)                                                                |
| [Federated Learning with Fair Averaging](https://arxiv.org/abs/2104.14937)                                                                                       |
| [Leveraging Sharing Communities to Achieve Federated Learning for Cybersecurity](https://arxiv.org/abs/2104.11763)                                               |
| [Communication-Efficient Federated Learning with Dual-Side Low-Rank Compression](https://arxiv.org/abs/2104.12416)                                               |
| [Communication-Efficient and Personalized Federated Lottery Ticket Learning](https://arxiv.org/abs/2104.12501)                                                   |
| [FedDPGAN: Federated Differentially Private Generative Adversarial Networks Framework for the Detection of COVID-19 Pneumonia](https://arxiv.org/abs/2104.12581) |
| [Semi-Decentralized Federated Edge Learning for Fast Convergence on Non-IID Data](https://arxiv.org/abs/2104.12678)                                              |
| [Simultaneous Wireless Information and Power Transfer for Federated Learning](https://arxiv.org/abs/2104.12749)                                                  |
| [Continual Distributed Learning for Crisis Management](https://arxiv.org/abs/2104.12876)                                                                         |
| [Confined Gradient Descent: Privacy-preserving Optimization for Federated Learning](https://arxiv.org/abs/2104.13050)                                            |
| [Towards On-Device Federated Learning: A Direct Acyclic Graph-based Blockchain Approach](https://arxiv.org/abs/2104.13092)                                       |
| [Secure and Efficient Federated Learning Through Layering and Sharding Blockchain](https://arxiv.org/abs/2104.13130)                                             |
| [A Graph Federated Architecture with Privacy Preserving Learning](https://arxiv.org/abs/2104.13215)                                                              |
| [Spatial Privacy-aware VR streaming](https://arxiv.org/abs/2104.14170)                                                                                           |
| [End-to-End Speech Recognition from Federated Acoustic Models](https://arxiv.org/abs/2104.14297)                                                                 |
| [From Distributed Machine Learning to Federated Learning: A Survey](https://arxiv.org/abs/2104.14362)                                                            |
| [PPFL: Privacy-preserving Federated Learning with Trusted Execution Environments](https://arxiv.org/abs/2104.14380)                                              |
| [Privacy-Preserving Federated Learning on Partitioned Attributes](https://arxiv.org/abs/2104.14383)                                                              |
| [On In-network learning. A Comparative Study with Federated and Split Learning](https://arxiv.org/abs/2104.14929)                                                |
| [FedProto: Federated Prototype Learning over Heterogeneous Devices](https://arxiv.org/abs/2105.00243)                                                            |
| [GRNN: Generative Regression Neural Network -- A Data Leakage Attack for Federated Learning](https://arxiv.org/abs/2105.00529)                                   |
| [Personalized Federated Learning by Structured and Unstructured Pruning under Data Heterogeneity](https://arxiv.org/abs/2105.00562)                              |
| [OCTOPUS: Overcoming Performance andPrivatization Bottlenecks in Distributed Learning](https://arxiv.org/abs/2105.00602)                                         |
| [Convergence Analysis and System Design for Federated Learning over Wireless Networks](https://arxiv.org/abs/2105.00872)                                         |
| [Citadel: Protecting Data Privacy and Model Confidentiality for Collaborative Learning with SGX](https://arxiv.org/abs/2105.01281)                               |

## 04.19 - 04.23
| Title                                                                                                                                                    |
| -------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Leveraging Sharing Communities to Achieve Federated Learning for Cybersecurity](https://arxiv.org/abs/2104.11763)                                       |
| [Practical Defences Against Model Inversion Attacks for Split Neural Networks](https://arxiv.org/abs/2104.05743)                                         |
| [Gradient Masked Federated Optimization](https://arxiv.org/abs/2104.10322)                                                                               |
| [Preventing and Controlling Epidemics through Blockchain-Assisted AI-Enabled Networks](https://arxiv.org/abs/2104.13224)                                 |
| [Personalized Semi-Supervised Federated Learning for Human Activity Recognition](https://arxiv.org/abs/2104.08094)                                       |
| [Research on Resource Allocation for Efficient Federated Learning](https://arxiv.org/abs/2104.09177)                                                     |
| [An Android App for Digital Transport Cards of Smart City: Request for Suggestions](https://arxiv.org/abs/2104.09245)                                    |
| [An Overview of Federated Learning at the Edge and Distributed Ledger Technologies for Robotic and Autonomous Systems](https://arxiv.org/abs/2104.10141) |
| [Auto-FedAvg: Learnable Federated Averaging for Multi-Institutional Medical Image Segmentation](https://arxiv.org/abs/2104.10195)                        |
| [Federated Traffic Synthesizing and Classification Using Generative Adversarial Networks](https://arxiv.org/abs/2104.10400)                              |
| [A Survey on Federated Learning and its Applications for Accelerating Industrial Internet of Things](https://arxiv.org/abs/2104.10501)                   |
| [Covert Channel Attack to Federated Learning Systems](https://arxiv.org/abs/2104.10561)                                                                  |
| [Privacy-Preserved Blockchain-Federated-Learning for Medical Image Analysis Towards Multiple Parties](https://arxiv.org/abs/2104.10903)                  |
| [Decentralized Federated Averaging](https://arxiv.org/abs/2104.11375)                                                                                    |
| [A study on Ensemble Learning for Time Series Forecasting and the need for Meta-Learning](https://arxiv.org/abs/2104.11475)                              |
| [Robust Federated Learning by Mixture of Experts](https://arxiv.org/abs/2104.11700)                                                                      |
| [Wireless Federated Learning (WFL) for 6G Networks -- Part II: The Compute-then-Transmit NOMA Paradigm](https://arxiv.org/abs/2104.12005)                |
| [FedSup: A Communication-Efficient Federated Learning Fatigue Driving Behaviors Supervision Framework](https://arxiv.org/abs/2104.12086)                 |
| [Identifying tax evasion in Mexico with tools from network science and machine learning](https://arxiv.org/abs/2104.13353)                               |

## 04.12 - 04.16


| Title                                                                                                                                                          |
| -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Resource Rationing for Wireless Federated Learning: Concept, Benefits, and Challenges](https://arxiv.org/abs/2104.06990)                                      |
| [FedGraphNN: A Federated Learning System and Benchmark for Graph Neural Networks](https://arxiv.org/abs/2104.07145)                                            |
| [PyVertical: A Vertical Federated Learning Framework for Multi-headed SplitNN](https://arxiv.org/abs/2104.00489)                                               |
| [CSAFL: A Clustered Semi-Asynchronous Federated Learning Framework](https://arxiv.org/abs/2104.08184)                                                          |
| [FedSAE: A Novel Self-Adaptive Federated Learning Framework in Heterogeneous Systems](https://arxiv.org/abs/2104.07515)                                        |
| [A Method to Reveal Speaker Identity in Distributed ASR Training, and How to Counter It](https://arxiv.org/abs/2104.07815)                                     |
| [BROADCAST: Reducing Both Stochastic and Compression Noise to Robustify Communication-Efficient Federated Learning](https://arxiv.org/abs/2104.06685)          |
| [Communication Efficient Federated Learning with Adaptive Quantization](https://arxiv.org/abs/2104.06023)                                                      |
| [D-Cliques: Compensating NonIIDness in Decentralized Federated Learning with Topology](https://arxiv.org/abs/2104.07365)                                       |
| [Efficient Ring-topology Decentralized Federated Learning with Deep Generative Models for Industrial Artificial Intelligent](https://arxiv.org/abs/2104.08100) |
| [FedCom: A Byzantine-Robust Local Model Aggregation Rule Using Data Commitment for Federated Learning](https://arxiv.org/abs/2104.08020)                       |
| [Federated Generalized Face Presentation Attack Detection](https://arxiv.org/abs/2104.06595)                                                                   |
| [Federated Learning for Internet of Things: A Comprehensive Survey](https://arxiv.org/abs/2104.07914)                                                          |
| [Federated Learning of User Verification Models Without Sharing Embeddings](https://arxiv.org/abs/2104.08776)                                                  |
| [Federated Learning-based Active Authentication on Mobile Devices](https://arxiv.org/abs/2104.07158)                                                           |
| [FedNLP: A Research Platform for Federated Learning in Natural Language Processing](https://arxiv.org/abs/2104.08815)                                          |
| [On the Importance of Trust in Next-Generation Networked CPS Systems: An AI Perspective](https://arxiv.org/abs/2104.07853)                                     |
| [Personalized Semi-Supervised Federated Learning for Human Activity Recognition](https://arxiv.org/abs/2104.08094)                                             |
| [Practical Defences Against Model Inversion Attacks for Split Neural Networks](https://arxiv.org/abs/2104.05743)                                               |
| [Privacy-preserving Federated Learning based on Multi-key Homomorphic Encryption](https://arxiv.org/abs/2104.06824)                                            |
| [Sample-based and Feature-based Federated Learning via Mini-batch SSCA](https://arxiv.org/abs/2104.06011)                                                      |
| [See through Gradients: Image Batch Recovery via GradInversion](https://arxiv.org/abs/2104.07586)                                                              |
| [Serverless Federated Learning for UAV Networks: Architecture, Challenges, and Opportunities](https://arxiv.org/abs/2104.07557)                                |
| [The Role of Cross-Silo Federated Learning in Facilitating Data Sharing in the Agri-Food Sector](https://arxiv.org/abs/2104.07468)                             |
| [Towards Causal Federated Learning For Enhanced Robustness and Privacy](https://arxiv.org/abs/2104.06557)                                                      |

## 04.05 - 04.09

| Title                                                                                                                                                  |
| ------------------------------------------------------------------------------------------------------------------------------------------------------ |
| [FedPandemic: A Cross-Device Federated Learning Approach Towards Elementary Prognosis of Diseases During a Pandemic](https://arxiv.org/abs/2104.01864) |
| [SCAFFOLD: Stochastic Controlled Averaging for Federated Learning](https://arxiv.org/abs/1910.06378)                                                   |
| [Federated Learning Meets Blockchain in Edge Computing: Opportunities and Challenges](https://arxiv.org/abs/2104.01776)                                |
| [Accelerated Gradient Tracking over Time-varying Graphs for Decentralized Optimization](https://arxiv.org/abs/2104.02596)                              |
| [Bayesian Variational Federated Learning and Unlearning in Decentralized Networks](https://arxiv.org/abs/2104.03834)                                   |
| [Bayesian Variational Federated Learning and Unlearning in Decentralized Networks](https://arxiv.org/abs/2104.03834)                                   |
| [Can Federated Learning Save The Planet?](https://arxiv.org/abs/2010.06537)                                                                            |
| [Communication-Efficient Agnostic Federated Averaging](https://arxiv.org/abs/2104.02748)                                                               |
| [Distributed Learning in Wireless Networks: Recent Progress and Future Challenges](https://arxiv.org/abs/2104.02151)                                   |
| [Empowering Prosumer Communities in Smart Grid with Wireless Communications and Federated Edge Learning](https://arxiv.org/abs/2104.03169)             |
| [Fast-adapting and Privacy-preserving Federated Recommender System](https://arxiv.org/abs/2104.00919)                                                  |
| [Fast-adapting and Privacy-preserving Federated Recommender System](https://arxiv.org/abs/2104.00919)                                                  |
| [Fast-Convergent Federated Learning with Adaptive Weighting](https://arxiv.org/abs/2012.00661)                                                         |
| [Federated Bandit: A Gossiping Approach](https://arxiv.org/abs/2010.12763)                                                                             |
| [FedFace: Collaborative Learning of Face Recognition Model](https://arxiv.org/abs/2104.03008)                                                          |
| [FedFace: Collaborative Learning of Face Recognition Model](https://arxiv.org/abs/2104.03008)                                                          |
| [FedGroup: Efficient Clustered Federated Learning via Decomposed Data-Driven Measure](https://arxiv.org/abs/2010.06870)                                |
| [FedMood: Federated Learning on Mobile Health Data for Mood Detection](https://arxiv.org/abs/2102.09342)                                               |
| [FL-AGCNS: Federated Learning Framework for Automatic Graph Convolutional Network Search](https://arxiv.org/abs/2104.04141)                            |
| [Flower: A Friendly Federated Learning Research Framework](https://arxiv.org/abs/2007.14390)                                                           |
| [Imagine All the People: Citizen Science, Artificial Intelligence, and Computational Research](https://arxiv.org/abs/2104.00093)                       |
| [Joint Optimization of Communications and Federated Learning Over the Air](https://arxiv.org/abs/2104.03490)                                           |
| [On-device Federated Learning with Flower](https://arxiv.org/abs/2104.03042)                                                                           |
| [Personalized Federated Learning: A Unified Framework and Universal Optimization Techniques](https://arxiv.org/abs/2102.09743)                         |
| [Privacy Leakage of Real-World Vertical Federated Learning](https://arxiv.org/abs/2011.09290)                                                          |
| [SecureBoost: A Lossless Federated Learning Framework](https://arxiv.org/abs/1901.08755)                                                               |
